src/chuchichaestli/metrics/backbones/swav.py

```diff
@@ -0,0 +1,397 @@
+# From https://github.com/stanis-morozov/self-supervised-gan-eval/blob/main/src/self_supervised_gan_eval/resnet50.py
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+import torch
+import torch.nn as nn
+import ssl
+import torch.nn.functional as F
+from torchvision.transforms import Normalize, InterpolationMode
+
+
+ssl._create_default_https_context = ssl._create_unverified_context
+
+# import lightly
+try:
+    from torchvision.models.utils import load_state_dict_from_url
+except ImportError:
+    from torch.utils.model_zoo import load_url as load_state_dict_from_url
+from torchvision import transforms
+
+from chuchichaestli.metrics.backbones.backbone_encoder import Encoder
+
+SWAV_WEIGHTS_URL = (
+    "https://www.dropbox.com/s/ghjrgktg0lyjorn/swav_800ep_pretrain.pth.tar?dl=1"
```

Comment:
Is this your Dropbox or someone else's? If it isn't yours, could you upload it to SwitchDrive or somewhere where we have more control over it?